recipes/unitig-caller/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "unitig-caller" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/johnlees/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: 041223cb6c08eae317ea8e73d94521f0dcaedee374afc5e3b2ce639695937a4a
+
+build:
+  number: 0
+  skip: True #  [osx]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - unitig-caller = unitig_caller.__main__:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python >=3
+    - pip
+    - cmake
+    - pybind11
```

Comment:
Can you try moving `cmake` to the `build:` section? `pybind11` only contains headers, correct?

Reply:
Yes, I believe `pybind11` is header only (so not in run). I will move `cmake`